src/mdtest.c

```diff
@@ -1573,7 +1573,9 @@ void valid_tests() {
         FAIL("-c not compatible with -B");
     }
 
-    if (strcasecmp(backend_name, "POSIX") != 0 && strcasecmp(backend_name, "DUMMY") != 0) {
+    if ((strcasecmp(backend_name, "POSIX") != 0) &&
+        (strcasecmp(backend_name, "IME")   != 0) &&
+        (strcasecmp(backend_name, "DUMMY") != 0)) {
       FAIL("-a only supported interface is POSIX (and DUMMY) right now!");
```

Comment:
Can you update the error message to include IME?

Reply:
Sure I'll resubmit a version. Thanks for the review!